### ISSUE TITLE
[FR] Integrate openAI into AppFlowy

### ISSUE THREAD

**Issue by LucasXu0** [AppFlowy-IO/AppFlowy#1752]
### Description

Please create a new issue before starting to develop the next tasks.

- [x] #1859
- [ ] Continue to Write
- [x] #1866
- [ ] ...

### Impact

The users who want to use openAI will benefit from this feature.

### Additional Context

I have implemented some AI plugins with simple UI / UX as an example by using openAI. Please refer to `AppFlowy/frontend/app_flowy/packages/appflowy_editor/example/lib/plugin/AI`.

**Comment by LucasXu0** [comment]
- [] do some research and write a plan for open AI
- [] Implemented 3 AI plugins in v0.1.1
